libs/exec/execv_windows.go

```diff
@@ -0,0 +1,95 @@
+//go:build windows
+
+package exec
+
+import (
+	"bufio"
+	"fmt"
+	"os"
+	"os/exec"
+	"strings"
+	"sync"
+)
+
+// Note: Windows does not support an execv syscall that replaces the current process.
+// Thus we emulate that by launching a child process and streaming the output and returning
+// the exit code.
+// ref: https://learn.microsoft.com/en-us/cpp/c-runtime-library/reference/execv-wexecv?view=msvc-170
+func execv(opts ExecvOptions) error {
+	path, err := exec.LookPath(opts.Args[0])
+	if err != nil {
+		return fmt.Errorf("looking up %q failed: %w", opts.Args[0], err)
+	}
+
+	cmd := exec.Command(path, opts.Args[1:]...)
+
+	cmd.Dir = opts.Dir
+	cmd.Env = opts.Env
+
+	stdout, err := cmd.StdoutPipe()
```

Comment:
Is it possible to avoid pipes + threads to read them and inherit the file descriptors?

Reply:
+1 -- you can use `os.Stdout` and `os.Stderr`.

Also `os.Stdin`, btw.

Reply:
This was my initial approach. I recall running into issues where the output was not completely flushed on windows and thus I had to revert back to the streaming approach in this commit:
https://github.com/databricks/cli/pull/2413/commits/be3be9cb2f1a212c5c881c6018dd6fbd0416b8d4

Note, we use the same approach in the experimental scripts section:
https://github.com/databricks/cli/blob/297059241f62c250c8b0cf45120804faeaab89ef/libs/exec/exec.go#L119

Note, there might be a way to make fd inheritance work on windows with proper flushing. I don't consider that to be blocking for this PR. Please let me know if you disagree.

Reply:
Note, I'm not endorsing SysProcAttr setting here, just pasting complete script as a starting point.

Reply:
Thanks, tried it out and just passing the file handles seems to work fine. Updated the PR.
```
cmd.Stdin = os.Stdin
cmd.Stdout = os.Stdout
cmd.Stderr = os.Stderr
```